conanfile.py

```diff
@@ -49,38 +55,39 @@ def package(self):
         self.copy("*.lib", dst="lib", src=".", keep_path=False)
 
     def package_info(self):
-        self.cpp_info.libs = ["absl_synchronization",                              
-                              "absl_symbolize",
-                              "absl_malloc_internal",
-                              "absl_time",
-                              "absl_strings",
+        self.cpp_info.libs = ["absl_any",

```

Comment:
maybe this new order is causing Travis error. Abseil has circular dependency from itself.

Reply:
definitely. the problem abseil has added some new libraries, and removed previously available, so we have to guess new order now.

Reply:
Thanks Alf! :joy_cat: